### PR TITLE
retryhttp client

### PR DESCRIPTION
**Pull request by maurafortino**
What's Included:
- updated v2.Send to use retryhttp client
- fixed a couple of spelling mistakes

### CODE REVIEW

File: internal/sink/sink.go [hunk 1]
```diff
@@ -215,29 +215,6 @@ func getUrls(urls []string) (int, error) {
 
 }
 
-func (whs WebhookSink) Send(secret, acceptType string, msg *wrp.Message) error {
```
Comment (maurafortino) [inline internal/sink/sink.go:218]: moved this down toward the v2 logic so that they are all together

File: internal/sink/sink.go [hunk 2]
```diff
@@ -650,22 +627,6 @@ func AddMessageHeaders(kafkaMsg *sarama.ProducerMessage, m *wrp.Message) {
 	}
 }
 
-func (v2 *WebhookV2) Update(c Config, l *zap.Logger, altUrls []string, id, failureUrl, receiverUrl string) {
```
Comment (maurafortino) [inline internal/sink/sink.go:653]: function wasn't being used

File: internal/sink/srvRecordDialer.go
Comment (maurafortino) [inline internal/sink/srvRecordDialer.go:1]: Note: we will be updating the logic for the LoadBalancingScheme in the next PR

**Review by denopink — APPROVED** [review]
LGTM! Keep up the great work! 🍻

**Pull request merged**